### ISSUE TITLE
`manuscript-cli help <cmd>` and `manuscript-cli <cmd> --help` produce runtime errors

### ISSUE THREAD

**Issue by KagemniKarimu** [chainbase-labs/manuscript-core#66]
### Self Checks

- [X] This is only for bug report, if you would like to ask a question, please head to [Discussions](https://github.com/chainbase-labs/manuscript-core/discussions).
- [X] I have searched for existing issues [search for existing issues](https://github.com/chainbase-labs/manuscript-core/issues), including closed ones.
- [X] I have searched for existing pull requests [search for existing pull requests](https://github.com/chainbase-labs/manuscript-core/pulls), including closed ones.

### Environment

> **Linux:** Linux fedora-workstation 6.11.10-300.fc41.x86_64 https://github.com/chainbase-labs/manuscript-core/pull/1 SMP PREEMPT_DYNAMIC Sat Nov 23 00:51:20 UTC 2024 x86_64 GNU/Linux
>     **Docker:** Docker version 27.3.1, build ce12230 , Docker Desktop 4.36.0 (175267)
>     **Manuscript-core:** Latest commit (as of 12/04/2024)

### Steps to reproduce

#  :exclamation: Case 1
Use `manuscript-cli help <cmd_name>` where `<cmd_name>` is any command on the CLI.

# :exclamation: Case 2
Use `manuscript-cli <cmd_name> --help` where `<cmd_name>` is any command on the CLI.

### ✔️ Expected Behavior

:green_circle: Running `manuscript-cli list --help`:
```
The list command shows all running, initializing, and recently stopped manuscript jobs.
EXAMPLE USAGE: manuscript-cli list
# rest of info goes here
```

:green_circle: Running `manuscript-cli help list`:
```
The list command shows all running, initializing, and recently stopped manuscript jobs.
EXAMPLE USAGE: manuscript-cli list
# rest of info goes here
```

### ❌ Actual Behavior

:warning: Running `manuscript-cli list --help`:
```
panic: runtime error: index out of range [4] with length 0

goroutine 1 [running]:
manuscript-core/commands.init.0.func1(0xdd0e60, {0xdc45f0?, 0xa70a80?, 0xdc45f0?})
	/home/kagemnikarimu/ChainbaseProjects/manuscript-core/commands/commandder.go:103 +0x75c
github.com/spf13/cobra.(*Command).ExecuteC(0xdd1fa0)
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1122 +0x5d7
github.com/spf13/cobra.(*Command).Execute(...)
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
manuscript-core/commands.Execute({0x0?, 0x409d45?, 0x4015f0?})
	/home/kagemnikarimu/ChainbaseProjects/manuscript-core/commands/commandder.go:16 +0x1a
main.main()
	/home/kagemnikarimu/ChainbaseProjects/manuscript-core/main.go:24 +0x56
```

:warning: Running `manuscript-cli help list`:
```
panic: runtime error: index out of range [4] with length 0

goroutine 1 [running]:
manuscript-core/commands.init.0.func1(0xdd0e60, {0xc0000f4d00?, 0xc0000f4d00?, 0xc0000f4d00?})
	/home/kagemnikarimu/ChainbaseProjects/manuscript-core/commands/commandder.go:103 +0x75c
github.com/spf13/cobra.(*Command).Help(0xdd0e60)
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:476 +0x34
github.com/spf13/cobra.(*Command).InitDefaultHelpCmd.func2(0xc0000fe308, {0xc000040450, 0x1, 0x1})
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1270 +0x68
github.com/spf13/cobra.(*Command).execute(0xc0000fe308, {0xc000040410, 0x1, 0x1})
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:989 +0xab1
github.com/spf13/cobra.(*Command).ExecuteC(0xdd1fa0)
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1117 +0x3ff
github.com/spf13/cobra.(*Command).Execute(...)
	/home/kagemnikarimu/go/pkg/mod/github.com/spf13/cobra@v1.8.1/command.go:1041
manuscript-core/commands.Execute({0x0?, 0x409d45?, 0x4015f0?})
	/home/kagemnikarimu/ChainbaseProjects/manuscript-core/commands/commandder.go:16 +0x1a
main.main()
	/home/kagemnikarimu/ChainbaseProjects/manuscript-core/main.go:24 +0x56
```

**Comment by KagemniKarimu** [comment]
I have a PR in the works which will solve this issue :fire:  ! @Liquidwe

**Comment by KagemniKarimu** [comment]
Issue was solved by #69 :smile:
